.gitignore

```diff
@@ -33,7 +33,6 @@ Engine/resource/version.aps
 *.o
 *.d
 common.a
-ags
```

Comment:
We need to have Engine/ags in ignore list, because it is produced by making Makefile directly (at least on Linux).